docs/README.md

```diff
@@ -34,6 +34,54 @@ You'll need a `kubectl`-style config file to connect to your cluster.
 `$HOME/.kube/config`. You can specify an alternate kubeconfig connection file
 with `--kubeconfig`, or the env var `$KUBECONFIG`, for any command.
 
+## Kn Config
+
+To better customize kn there is a set of configuration parameters you are able to setup and customize. In particular you can specify where your kn plugins are located and how they are found. The kn configuration file is meant to capture these configuration options. Let's explore this file, it's location, and the options you are able to change with it.
+
+### Location
+
+By default the kn config file is located in the user's home directory under `.kn` hidden subdirectory and is named `config.yaml`. So in `~/.kn/config.yaml`. It is not created for you by default. You can easily create one with:
```

Comment:
The default location kn looks for config is under the home directory of user at `$HOME/.kn/config.yaml`. It is not created for you part of Kn installation. You can create this file elsewhere and use `--config` flag to specify its path.

Reply:
I'd just mention the location description and let `options` section describe the contents of it, the rest of the below contents in `### Location` section can be removed IMO.

Reply:
OK I simplified it. Kept the location but reduced. See latest version if it reads better.